src/pivotal-ui/javascripts/pivotal-ui.js

```diff
@@ -7,3 +7,4 @@ require('pui-prismjs');
 require('./scale')();
 require('@npmcorp/pui-react-back-to-top/jquery-plugin');
 require('@npmcorp/pui-css-drop-down-menu');
+require('../components/forms');
```

Comment:
It will still work after publication that way? What's `../components/forms` in that context?